Fix vale warnings in End-user Guide (#2078)
codenvy/che-docs: modules/end-user-guide/partials/ref_che-theia-plug-in-metadata.adoc

            * autoMigrate - boolean 
            
            * migrateTo - new `org/plugin-id/version`, for example `redhat/vscode-apache-camel/latest`

> |`firstPublicationDate`| Not required to be in YAML; if it is not included, it will be generated during Plugin Registry dockerimage build

Reviewer: ```suggestion
|`firstPublicationDate`| Not required. If it is not present in the YAML, the Plugin Registry dockerimage build generates it.
```
